### PR TITLE
Pull Request for the Restaurant Page v1.1 project for Javascript module 

### PR DESCRIPTION
**Pull request by jose-Abel**
Hello,

This is my request for a Pull Review for the Restaurant Page v1.1 project for Javascript module that consist of creating a multi page websites with Javascript and webpack where all the HTML elements had to be created with Javascript. 

Hoping to get an amazing feedback!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "restaurantpage",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (bellom) [inline package.json]: We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

We also need inform users about these script in the README file. This information can be placed in the Usage section. This way users can know the command to run to start your application.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jose-Abel  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a 1 issue left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_*

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const h1 = document.createElement('h1');
+  const onlyImage = new Image(500, 500);
+
+  // Styles to the HTML elements
+  body.style.background = 'linear-gradient(to right, #64f38c, #f79d00)';
+  body.style.fontFamily = "'Pattaya', sans-serif";
+  contentId.style.margin = '0 3rem;';
+  h1.style.textAlign = 'center';
+  h1.style.color = '#fceabb';
+  h1.style.fontSize = '5rem';
```
Comment (rindrajosia) [inline src/contact.js:16]: Keep all the styling logic in CSS files

File: src/divBox.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  const div1 = document.createElement('div');
+  const div2 = document.createElement('div');
+  const span1 = document.createElement('span');
+  const span2 = document.createElement('span');
+
+  // Adding styles to the HTML elements
+  div1.style.display = 'flex';
+  div1.style.margin = '4rem 0';
+  div1.style.borderRadius = '5rem';
+  div1.style.border = '5px solid #fceabb';
```
Comment (rindrajosia) [inline src/divBox.js:14]: Keep all the styling logic in CSS files

File: src/divBox.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  if (imageElement.style.width !== '200px') {
+    imageElement.style.width = '500px';
+    imageElement.style.height = '500px';
+    imageElement.style.borderRadius = '20%';
```
Comment (rindrajosia) [inline src/divBox.js:19]: Keep all the styling logic in CSS files

File: src/divBox.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  if (h3Element) {
+    h3Element.style.textAlign = 'center';
+    h3Element.style.color = '#fceabb';
+    h3Element.style.fontSize = '2rem';
```
Comment (rindrajosia) [inline src/divBox.js:25]: Keep all the styling logic in CSS files

File: src/divBox.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  div2.style.display = 'flex';
+  div2.style.flexDirection = 'column';
+  div2.style.lineHeight = '2';
+  div2.style.fontSize = '1.5rem';
+  div2.style.justifyContent = 'space-around';
+  span1.style.marginLeft = '1.5rem';
+  span2.style.marginLeft = '1.5rem';
```
Comment (rindrajosia) [inline src/divBox.js:34]: Keep all the styling logic in CSS files

File: src/home.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  body.style.background = 'linear-gradient(to right, #64f38c, #f79d00)';
+  body.style.fontFamily = "'Pattaya', sans-serif";
+  contentId.style.margin = '0 3rem;';
+  h1.style.textAlign = 'center';
+  h1.style.color = '#fceabb';
+  h1.style.fontSize = '5rem';
+  imagesArray[1].style.order = 1;
+  imagesArray[1].style.width = '200px';
+  imagesArray[1].style.height = '200px';
+  imagesArray[1].style.borderRadius = '20%';
```
Comment (rindrajosia) [inline src/home.js:31]: Keep all the styling logic in CSS files

File: src/menu.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+  contentId.style.margin = '0 3rem;';
+  h1.style.textAlign = 'center';
+  h1.style.color = '#fceabb';
+  h1.style.fontSize = '5rem';
+  h2.style.textAlign = 'center';
+  h2.style.color = '#fceabb';
+  h2.style.fontSize = '3.5rem';
+  secondH2.style.textAlign = 'center';
+  secondH2.style.color = '#fceabb';
+  secondH2.style.fontSize = '3.5rem';
```
Comment (rindrajosia) [inline src/menu.js:53]: Keep all the styling logic in CSS files

File: src/menu.js [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+  for (let i = 1; i < imagesArray.length; i += 2) {
+    imagesArray[i].style.order = 1;
```
Comment (rindrajosia) [inline src/menu.js:56]: Keep all the styling logic in CSS files

File: src/navbar.js [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  ul.style.display = 'flex';
+  ul.style.justifyContent = 'center';
+
+  for (let i = 0; i < liArray.length; i += 1) {
+    liArray[i].style.marginRight = '2rem';
+    liArray[i].style.listStyle = 'none';
+    liArray[i].style.fontSize = '2rem';
+    liArray[i].style.border = '4px solid #fceabb';
+    liArray[i].style.padding = '0 5px';
+    liArray[i].style.color = '#fceabb';
```
Comment (rindrajosia) [inline src/navbar.js:24]: Keep all the styling logic in CSS files

File: src/navbar.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+    aArray[i].type = 'button';
+    aArray[i].style.cursor = 'pointer';
```
Comment (rindrajosia) [inline src/navbar.js:27]: Keep all the styling logic in CSS files

File: src/navbar.js [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+  const li3 = document.createElement('li');
+  const a1 = document.createElement('a');
+  const a2 = document.createElement('a');
+  const a3 = document.createElement('a');
+  const liArray = [li1, li2, li3];
+  const aArray = [a1, a2, a3];
+
+  // Adding styles to the HTML elements
+  ul.style.display = 'flex';
+  ul.style.justifyContent = 'center';
```
Comment (rindrajosia) [inline src/navbar.js:16]: Keep all the styling logic in CSS files

**Review by rindrajosia — APPROVED** [review]
Approved! :tada:

Congratulations are in order!! :clap: Keep up the good work!

- [x] [OPTIONAL] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)
- [x] [OPTIONAL] Please check your live demo because the `navbar` doesn't work.

All the best on your next projects! :wink:
Rindra Josia

**Pull request merged**